packages/compass-e2e-tests/helpers/commands/connection-workspaces.ts

```diff
@@ -0,0 +1,34 @@
+import { TEST_MULTIPLE_CONNECTIONS } from '../compass';
+import type { CompassBrowser } from '../compass-browser';
+import * as Selectors from '../selectors';
+
+export async function navigateToConnectionTab(
+  browser: CompassBrowser,
+  connectionName: string,
+  tabName: 'Performance' | 'Databases'
+): Promise<void> {
+  if (TEST_MULTIPLE_CONNECTIONS) {
+    if (tabName === 'Databases') {
+      await browser.clickVisible(Selectors.sidebarConnection(connectionName));
+    } else {
+      // TODO(COMPASS-8002): click the three dots menu then the relevant option. (View performance metrics)
```

Comment:
We technically have everything to do this now in this PR, but I had to draw the line somewhere :)